cli/cmd/testdata/deployment_test_file.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Using similar logic, this should be named "TestInjectYAML_emojivoto_deployment.input.yml". In particular it is useful to have "input" in the file name.